private/buf/cmd/buf/command/generate/generate.go

```diff
@@ -390,10 +392,15 @@ func run(
 			return err
 		}
 	}
+	wasmPluginExecutor, err := bufwasm.NewPluginExecutor(ctx, filepath.Join(container.CacheDirPath(), "wasmplugins"))
```

Comment:
We have a top-level construct that handles the creation of all cache dirs, I can't remember where, @pkwarren will know

Reply:
So far cache dirs are only for the module cache, see bufcli's `createCacheDirs` as the current function that is used to create the cache dirs.

Reply:
I also think this should indicate it is the wasm plugins compilation cache. At first glance, I thought this might be a location where we cached remote wasm plugins for execution.